### PR TITLE
Added monster sort method for main page

### LINKED ISSUE #6: Categorization on main page
Main page needs to be sortable by monster "intention" category.

### PR DESCRIPTION
**Pull request by ashleycoxley**
Resolves #6 

@harperj

### CODE REVIEW

File: static/monsterSort.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	$('select').change(function() {
+		selection = $(this).val();
```
Comment (harperj) [inline static/monsterSort.js]: "selection" isn't a very useful name.  What type of thing is the value?  Monster-type?

File: static/monsterSort.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+		if (selection == 'all') {
+			$('.monster-box').show();
+		} else {
+			var selector = '.' + selection;
```
Comment (harperj) [inline static/monsterSort.js]: Might want to mention somewhere in a comment that the values are the same as class names on the monster boxes.

Comment (harperj) [inline static/monsterSort.js]: What happens if something else on the page had a `.good` class?

File: templates/main.html [hunk 3]
```diff
@@ -1,13 +1,38 @@
...
 
+<script src="{{url_for('static', filename='monsterSort.js')}}"></script>
 <div class='container-fluid monster-display-grid'>
 	<div class='row'>
+		<div class='col-md-12'>
+			<form id='monster-sort' class='form'>
+				<div class='form-group'>
+					<div class="col-sm-4 col-sm-offset-8">
+					  	<select class='form-control' name='monster-sort'>
+					  		<option selected='selected' disabled='disabled'>Sort by monster type</option>
```
Comment (harperj) [inline templates/main.html]: Indentation.

File: static/monsterSort.js (hunk 1 above)
Comment (ashleycoxley) [inline static/monsterSort.js]: k

File: static/monsterSort.js (hunk 2 above)
Comment (ashleycoxley) [inline static/monsterSort.js]: should i change it to ".monster-type-[good|evil|conflicted]" or something like that?

File: templates/main.html (hunk 3 above)
Comment (ashleycoxley) [inline templates/main.html]: k

File: static/monsterSort.js (hunk 2 above)
Comment (harperj) [inline static/monsterSort.js]: That's one option -- or you could use a compound selector like `.monster-box.good`.

Comment (ashleycoxley) [inline static/monsterSort.js]: oh, i see. Thank you!

**Pull request merged**